clippy_lints/src/float_literal.rs

```diff
@@ -109,29 +109,41 @@ impl<'tcx> LateLintPass<'tcx> for FloatLiteral {
                     // If the type suffix is missing the suggestion would be
```

Comment:
Looks like this comment here was meant to be part of the `if` that was moved down